lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -100,35 +109,30 @@ testRelease:
   variables:
     BUILD_TARGET: testRelease
```

Comment:
I think I needed to redefine the BUILD_TARGETs in some of these jobs to get multiple flavors to build. We may want to document that somewhere.

Reply:
I.e., `BUILD_TARGET: "testFlavor1Debug testFlavor2Debug"`

Reply:
Good point. Will update README.